.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -0,0 +1,145 @@
+---
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage-needed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: cosmos-version
+    attributes:
+      label: Astronomer Cosmos Version
+      description: What version of Astronomer Cosmos are you running? If you do not see your version in the list, please (ideally) test on
+        the latest release or main to see if the issue is fixed before reporting it.
+      options:
+        - "1.4.1"
+        - "main (development)"
+        - "Other Astronomer Cosmos version (please specify below)"
+      multiple: false
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: If "Other Astronomer Cosmos version" selected, which one?
+      # yamllint disable rule:line-length
+      description: >
+        On what version of Astronomer Cosmos are you currently experiencing the issue? Remember, you are encouraged to
+        test with the latest release or on the main branch to verify your issue still exists.
+  - type: input
+    id: dbt-core-version
+    attributes:
+      label: dbt-core version
+      description: What version of dbt-core are you running?
+      placeholder: ex. 1.8.0
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Versions of dbt adapters
+      description: What dbt adapter versions are you using?
+      placeholder: You can use `pip freeze | grep dbt` (you can leave only relevant ones)
+  - type: input
+    id: airflow-version
+    attributes:
+      label: airflow version
+      description: What version of Apache Airflow are you running?
+      placeholder: ex. 2.9.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating System
+      description: What Operating System are you using?
+      placeholder: "You can get it via `cat /etc/os-release` for example"
+    validations:
+      required: true
```

Comment:
This is probably not relevant - I don't remember us having an issue OS-specific so far..!

Reply:
Fair enough, we can keep it - we can always revisit this in the future!

Reply:
yes, thank you for reviewing and highlighting this.